.eslintignore

```diff
@@ -1,12 +1,10 @@
-bin/**
-build/**
-conf/**
-coverage/**
-docs/**
-scripts/**
-templates/**
-tests/bench/**
-tests/fixtures/**
-tests/performance/**
-tmp/**
-test.js
```

Comment:
I have `test.js` :)
It's included in `.gitignore` as well, so it's useful to verify issues.